src/components/tuits/tuit.js

```diff
@@ -50,11 +50,13 @@ const Tuit = ({tuit, deleteTuit, likeTuit, dislikeTuit}) => {
                 <Link to={`/tuit/${tuit._id}`}>
```

Comment:
This link worked before. It doesn't work now.

Reply:
The useEffect in tuit-screen somehow doesn't work, and I can't find out why. I don't think I touch the tuit-screen... Do you have any idea why the useEffect doesn't render?